apps/wallet/src/ui/index.tsx

```diff
@@ -75,9 +76,11 @@ function AppWrapper() {
 							}}
 						>
 							<SuiClientProvider networks={{ [api.apiEnv]: api.instance.fullNode }}>
-								<ErrorBoundary>
-									<App />
-								</ErrorBoundary>
+								<AccountsFormProvider>
```

Comment:
Ended up moving this here because I used the context in the create account mutation